packages/cozy-client/src/models/index.js

```diff
@@ -6,6 +6,7 @@ import * as folder from './folder'
 import * as account from './account'
 import * as note from './note'
 import * as permission from './permission'
+import * as utils from './utils'
```

Comment:
I don't think there is a utils model?

Reply:
https://github.com/cozy/cozy-client/pull/738/files#diff-3be6cca06b5c09545418c6c909dc9d87 ?

Reply:
I missed the second commit 🤦‍♂️